ocs_ci/ocs/ui/base_ui.py

```diff
@@ -376,6 +376,8 @@ def __init__(self, driver):
                 self.storage_class = "managed-csi_sc"
             else:
                 self.storage_class = "managed-premium_sc"
+        elif config.ENV_DATA["platform"].lower() == constants.GCP_PLATFORM:
```

Comment:
When reading from config, i would suggest to use `get.("key", default_value)` approach

Reply:
Please open a new issue because I did not use your method on all sc [thin, gp2...]